src/ensembl/stories/shared-components/chevron/Chevron.stories.tsx

```diff
@@ -0,0 +1,140 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+
+import Chevron, {
+  Direction as ChevronDirection
+} from 'src/shared/components/chevron/Chevron';
+import RadioGroup from 'src/shared/components/radio-group/RadioGroup';
+
+import styles from './Chevron.stories.scss';
+
+export const ChevronStory = () => {
+  const [direction, setDirection] = useState('down');
+  const [animation, setAnimation] = useState(false);
+  const [customClass, setCustomClass] = useState(false);
+
+  const directionOptions = [
+    {
+      value: 'down',
+      label: 'Down'
+    },
+    {
+      value: 'up',
+      label: 'Up'
+    },
+    {
+      value: 'left',
+      label: 'Left'
+    },
+    {
+      value: 'right',
+      label: 'Right'
+    }
+  ];
+
+  const animationOptions = [
+    {
+      value: false,
+      label: 'No animation'
+    },
+    {
+      value: true,
+      label: 'With animation'
+    }
+  ];
+
+  const classOptions = [
+    {
+      value: false,
+      label: 'Default'
+    },
+    {
+      value: true,
+      label: 'Custom'
+    }
+  ];
+
+  return (
+    <div className={styles.storyWrapper}>
+      <div className={styles.showRoom}>
+        <span className={styles.label}>Some text next to a chevron</span>
+        <Chevron
+          direction={direction as ChevronDirection}
+          animateDirectionChange={animation}
+          classNames={{ svg: customClass ? styles.customChevron : undefined }}
+        />
+      </div>
+      <div className={styles.controls}>
+        <div className={styles.controlsDirection}>
+          Direction
+          <RadioGroup
+            options={directionOptions}
+            selectedOption={direction}
+            onChange={(direction) => setDirection(direction as string)}
+          />
+        </div>
+        <div className={styles.controlsAnimation}>
+          Animation
+          <RadioGroup
+            options={animationOptions}
+            selectedOption={animation}
+            onChange={(animation) => setAnimation(animation as boolean)}
+          />
+        </div>
+        <div className={styles.controlsClass}>
+          CSS class
+          <RadioGroup
+            options={classOptions}
+            selectedOption={customClass}
+            onChange={(customClass) => setCustomClass(customClass as boolean)}
+          />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+ChevronStory.storyName = 'default';
+
+export const ChevronWrappedInButtonStory = () => {
+  const [direction, setDirection] = useState<ChevronDirection>('down');
+
+  const handleClick = () => {
+    const newDirection = direction === 'up' ? 'down' : 'up';
+    setDirection(newDirection);
+  };
+
+  return (
+    <div className={styles.storyWrapper}>
+      <div className={styles.showRoom}>
+        <span className={styles.label}>{'Click the chevron →'}</span>
```

Comment:
Why do we need a JSX expression for 'Click the chevron →' ?

Reply:
✅